components/Page.js

```diff
@@ -0,0 +1,22 @@
+import Link from "next/link";
+import Router from "next/router";
+import handleClick from "./activeLink";
+const Header = () => {
+  // <body background="https://i.pinimg.com/originals/4a/aa/a3/4aaaa35e178c1cc7c300c53556077d63.jpg">
+  return (
+    <>
+      <title>TribeBytes</title>
```

Comment:
This should be a part of the Next Head component. It can't be a part of the body. Check the documentation at https://nextjs.org/docs/api-reference/next/head